index.js

```diff
@@ -271,7 +282,37 @@ function logError(context, error){
             return null;
         }
       } while (nbResults === pageSize);
-  
+
+    if (version >= "8.0" && !data.noSecurityHotspot) {
+      do {
+        try {
+            const response = await got(`${sonarBaseURL}/api/hotspots/search?projectKey=${data.projectName}&ps=${pageSize}&p=${page}&statuses=TO_REVIEW`, {
```

Comment:
there may be more statuses than "TO_REVIEW" that are of interest. Can you list all the status in the conversation? there is probably a status like "CONFIRMED" or something that we definitely want to export?

Reply:
Possible status for api/hotspots/search API are "TO_REVIEW" and "REVIEWED"

Reply:
I propose to define a new HOTSPOT_STATUSES variable

Reply:
you are right it's only those 2 statuses